src/businesses/service/BusinessesService.ts

```diff
@@ -0,0 +1,37 @@
+import Business from '../model/Business';
+import RequestClientWrapper from '../../RequestClientWrapper';
+import { AxiosRequestConfig, AxiosResponse } from 'axios';
+import Resource from '../../model/Resource';
+import Version from '../../model/Version';
+import GenerateApiError from '../../utils/GenerateApiError';
+import * as Q from 'q';
+
+const EXPECTED_GET_STATUS = 200;
+
+export default class BusinessesService {
+
+  constructor(private readonly client: RequestClientWrapper) { }
+
+  public get(userId: string): Promise<Business[]> {
```

Comment:
You should also add support to load a single business:
https://docs.channelape.io/#daaa44ed-e6b6-47cc-9b1d-dba38908f2f5